README.md

```diff
@@ -43,6 +42,8 @@ services:
             - 443:443/tcp
         environment:
             - CERTBOT_EMAIL=owner@company.com
+        volumes:
+          - ./conf.d:/etc/nginx/user.conf.d
```

Comment:
Let's put `:ro` at the end of this, to ensure that user configuration is never clobbered by the scripts in this image.